resources/fixtures/American_DJ/American-DJ-Auto-Spot-150.qxf

```diff
@@ -21,15 +21,15 @@
   <Capability Min="80" Max="95" Preset="GoboMacro" Res1="Others/gobo00032.svg">Gobo 5</Capability>
   <Capability Min="96" Max="111" Preset="GoboMacro" Res1="Robe/gobo00037.svg">Gobo 6</Capability>
   <Capability Min="112" Max="127" Preset="GoboMacro" Res1="SGM/gobo00027.svg">Gobo 7</Capability>
-  <Capability Min="128" Max="255" Preset="GenericPicture" Res1="Others/rainbow.png">Rotating slow-fast</Capability>
+  <Capability Min="128" Max="255" Preset="RotationClockwiseSlowToFast">Rotating slow-fast</Capability>
  </Channel>
  <Channel Name="Gobo Rotation">
   <Group Byte="0">Speed</Group>
```

Comment:
This was 'Gobo' in another definition

Reply:
The type of group used for gobo rotation channels varies depending on who created the fixture. Up until now we have considered both "Gobo" or "Speed" as valid. We can make this consistently "Gobo" as well, but then all newly added devices must be checked for consistency. Please advise...

Reply:
Let's stick to "Gobo" as well as "Prism rotation" is set to "Prism"